.github/workflows/coverage.yml

```diff
@@ -0,0 +1,71 @@
+name: Coverage
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # Check all PR
+
+concurrency:
+  group: coverage-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+jobs:
+  coverage:
+    runs-on: ubuntu-22.04
+    name: collect code coverage
+    steps:
+      - name: free disk space
+        run: sudo rm -rf /usr/share/dotnet /usr/local/lib/android || true
```

Comment:
This might not be required here and should save 1 min of CI time. I think we only need it when running the C++ tests.

Reply:
Might be. I give it a try. I love it that nobody knows anymore by heart what the CI is doing 😅